doc/user-guide/06-layout-configuration.adoc

```diff
@@ -136,7 +136,7 @@ exclude the components on the disk from the layout file.
 A generic mechanism for doing this is the +EXCLUDE_RECREATE+
 variable. It can contain a list of storage components that will be
 excluded from the layout (will be present in the layout file, but
-commented out) recursively together with their descendands (components
+commented out) recursively together with their descendants (components
```

Comment:
How is it related to the topic of the PR (autodetecting secure boot)?

Reply:
Not at all, but it removes a blocker from merging. Happy to remove that if fixed elsewhere

Reply:
OK, please do so and I'll remove it from my branch

Reply:
done - b9814da104964fe046dbea8fbe68c95dc966446e